docs/5 IBC and Relayer/1 Relayer.md

```diff
@@ -0,0 +1,23 @@
+# Relayer
+
+Starport has a built-in IBC relayer that let's you connect local (running on you local machine) and remote blockchains. Starport relayer uses [IBC Go relayer](https://github.com/cosmos/relayer).
+
+## `starport relayer configure`
+
+`configure` command configures a connection between two blockchains. You will be prompted for RPC endpoints (required) and faucet endpoints (optional). Accounts used by the relayer are created on both blockchains and faucets are used (if available) to automatically fetch tokens. If the relayer fails to recieve tokens from a faucet, you will need to manually send tokens to addresses. By default a connection is setup for the `ibc-transfer` module for token transfers.
+
+Optional `--advanced` flag: let's you configure port and version for custom IBC module.
+
+By default configuration is stored in `$HOME/.relayer/`.
+
+## Example
+
+All values can be passed with flags.
```

Comment:
```suggestion
All flags are optional. All values can be passed with flags.
```

Reply:
I am the type of learner who enjoys a narrative that goes along with a technical command.

Reply:
Yes, I'll add description in a separate PR.